MaterialDesignThemes.UITests/WPF/TextFieldDefaultHeightTests.cs

```diff
@@ -30,22 +29,38 @@ public async Task SameHeightWithDefaultStyle()
     }
 
     [Fact]
-    public async Task SameHeightWithFloatingHintStyle()
+    public async Task SameHeightWithDefaultStyle_PostSmartHintRefactor()
     {
         await using var recorder = new TestRecorder(App);
 
+        // TODO: Add controls here as they adopt the new SmartHint approach. Once all controls have migrated, collapse into a single test with the old name.
         var stackPanel = await LoadXaml<StackPanel>(@"
 <StackPanel>
-    <TextBox materialDesign:HintAssist.IsFloating=""True"" />
-    <PasswordBox materialDesign:HintAssist.IsFloating=""True"" />
-    <ComboBox materialDesign:HintAssist.IsFloating=""True"" />
-    <DatePicker materialDesign:HintAssist.IsFloating=""True"" />
-    <materialDesign:TimePicker materialDesign:HintAssist.IsFloating=""True"" />
+    <TextBox />
 </StackPanel>");
 
         var height = await GetHeight(stackPanel, "TextBox");
+        //Assert.Equal(height, await GetHeight(stackPanel, "ComboBox"), Precision);
```

Comment:
Comment left here simply to indicate what is described below:

In the other PR's adopting the new `SmartHint` you will see controls move from the original test method into the `Xxx_PostSmartHintRefactor()` method (one control per PR). An example is the ComboBox, where you can see this happen:
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/3515/files#diff-e858ebe6ad71bec638332cb3f89c65f1570e5a3bcf92985e967c1ccb74f094d3

Once all styles have adopted the new approach, we can collapse into a single test with the original name again.